### ISSUE TITLE
Changing the Admin Email on Webonary

### ISSUE THREAD

**Issue by vernastutzman** [sillsdev/webonary#104]
![Changing Admin Email on Webonary](https://user-images.githubusercontent.com/53490028/86631012-36e07980-bf93-11ea-9ce2-20929057e474.JPG)

This is done under General Settings. When changing the Admin email for a user site, the new email needs to be confirmed. Apparently, the user never received an email from Webonary to confirm the email change. So the change is never made.

**Comment by phillip-hopper** [comment]
Check if the email is being sent with the correct authentication and signing.

**Comment by vernastutzman** [comment]
![Screenshot AdminEmailChange](https://user-images.githubusercontent.com/53490028/174901338-d108d1da-c1fb-4a93-b2fb-3e7fe5d3356e.jpg)
I tested this on https://www.webonary.work/loudictionary/wp-admin/options-general.php, by changing the Admin email from verna_stutzman@sil.org to verna@stutzman.net
According to the note, I should have received a confirmation email at verna@stutzman.net.

I waited 30 minutes, but did not receive an email. I did check Spam and Trash folders.
